### ISSUE TITLE
Have you had any luck with x86 linux VM's?

### ISSUE THREAD

**Issue by WheezyE** [Itai-Nelken/RPi-QEMU-VM-scripts#5]
Hey, I was wondering if you had experimented with any Linux x86 qemu VM's on RPi4 while you were making your Win98 and MacOS VM's.  I've been trying to make an x86 Linux VM but can't seem to get one running and was wondering if you might have any insight.

Long story about my attempts:
I followed [this guide on getting qemu VM's with KVM working on Pi4](https://linuxhint.com/enable_kvm_kernel_module_on_raspberry_pi_os/), but can only get an aarch64 VM working. x86 doesn't seem to work.

Any x86 or x86-64 VM's (including Debian iso's and Alpine Linux iso's) give me errors when booting that look like this:
![image](https://user-images.githubusercontent.com/29649598/120236153-dbf1e980-c218-11eb-9a7a-07a6c5e86207.png)

I've tried qemu-system ("qemu") v5.2 packages by updating my Pi4 apt repos to include newer sid/testing packages:
```
# Add testing repo to sources list. This way we can download a newer qemu packages (I found I could get newer qemu-user-static packages this way for some previous experiments w/ those)
echo -E "deb http://http.us.debian.org/debian/ testing non-free contrib main" | sudo tee -a /etc/apt/sources.list.d/raspi.list >/dev/null
wget -O - https://ftp-master.debian.org/keys/archive-key-10.asc | sudo apt-key add - # "Active Signing Key" found here https://ftp-master.debian.org/keys.html
sudo apt-get update
# It would now be a really good idea to remove the testing repo from your sources list!

sudo apt install qemu qemu-user qemu-user-static
```
(although soon after that I accidentally borked by pi by updating my entire system with testing packages)
I'm going to try newly build qemu-system packages soon from your deb and report back. (after I uh re-flash my pi lol)

Here's my [Alpine Linux](https://alpinelinux.org/downloads/) aarch64 qemu VM image (which was installed from their "virtual" iso): [download link](https://mega.nz/file/0YpXjAhA#rz92clnK_vfWwswZpwXOQkrJd_awFMVcjJmmO3ltkfI)  You might have to enable KVM on the Pi4 to use this image? (according to the first couple steps of [that guide](https://linuxhint.com/enable_kvm_kernel_module_on_raspberry_pi_os/)).
```
# DISABLE THE TESTING APT REPO IF YOU ENABLED IT!
sudo apt update && sudo apt full-upgrade -y
uname -r # should say "#.#.##-v7l+"
echo "arm_64bit=1" | sudo tee -a /boot/config.txt
sudo reboot
uname -r # should say "#.#.##-v8+" for 64bit kernel
sudo apt install virt-manager libvirt0 qemu-system
sudo usermod -aG libvirt-qemu $(whoami)
sudo reboot
sudo virsh net-start default
sudo virsh net-autostart default
```

**Comment by Itai-Nelken** [comment]
I tried the gparted live cd and a few editions of puppy linux. the gparted cd didn't boot, and one of the editions of puppy linux I tried (can't remember which) did seem to boot but after leaving it for ~1 hour I gave up on getting it to boot...
it seems that no modern OS's work. the newest OS I got to boot was MacOS X 10.0 but its so slow that it looks like a fast slideshow, I read somewhere that newer versions where faster, but I couldn't get a newer version to boot.
me and @Reyansh-Khobragade tried Windows XP but while it worked on a x86 computer when setting it up, I just couldn't get it to boot on the pi. @Reyansh-Khobragade got it to boot in safe mode.

from my testing so far I have concluded that the pi4 just isn't powerful enough to run modern OS's. maybe a old version of linux will work.
